opm/simulators/wells/StandardWell_impl.hpp

```diff
@@ -666,12 +690,15 @@ namespace Opm
 
         }
 
-
         // add vol * dF/dt + Q to the well equations;
         for (int componentIdx = 0; componentIdx < numWellConservationEq; ++componentIdx) {
             // TODO: following the development in MSW, we need to convert the volume of the wellbore to be surface volume
             // since all the rates are under surface condition
-            EvalWell resWell_loc = (wellSurfaceVolumeFraction(componentIdx) - F0_[componentIdx]) * volume / dt;
+            EvalWell resWell_loc(numWellEq_ + numEq, 0.0);
+            if (FluidSystem::numActivePhases() > 1) {
```

Comment:
Maybe it is good to put `FluidSystem::numActivePhases() > 1`  in a function has meaningful function name. 

Because `FluidSystem::numActivePhases() > 1` is used in multiple places.